### ISSUE TITLE
language not used correctly for plant search

### ISSUE THREAD

**Issue by kitzbergerg** [ElektraInitiative/PermaplanT#433]
When searching for plants the language displayed in the frontend should match the language the browsers language.

## Steps to Reproduce the Problem

Set your browsers settings and set the default language to english.
Go to <https://pr.permaplant.net/seeds/new> and log in.
Search for a plant like `Prunus`. The name in brackets should now be english:
![image](https://github.com/ElektraInitiative/PermaplanT/assets/33102273/bd5f5072-02b4-4fea-ba72-98a35afba581)

Now set the language to german and do the same again (you might have to refresh the cache for it to load in german).
The name is brackets is still in english:
![image](https://github.com/ElektraInitiative/PermaplanT/assets/33102273/2ab0634a-3a51-428d-9461-1f3ea7e52ace)

## Expected Result

The name in brackets should be german instead.

## Actual Result

The name in brackets is english no matter the language set in the browser.

## How to fix

When searching for plants the following will be returned from the backend:
```json
{
    "results": [
        {
            "id": 4572,
            "unique_name": "Prunus padus",
            "common_name_en": [
                "bird cherry"
            ]
        },
        {
            "id": 7000,
            "unique_name": "Prunus mume",
            "common_name_en": [
                "japanese apricot"
            ]
        },
        ...
    ],
    "page": 1,
    "per_page": 30,
    "total_pages": 13
}
```

As you can see only the `common_name_en` is returned (which is used to set the name in the brackets). You therefore need to go to the backend and extend the `PlantsSummaryDto` to include the `common_name_de` as well in the response.

In the frontend you then need to check the language. If it is german use `common_name_de` otherwise use `common_name_en`.

**Comment by markus2330** [comment]
Thanks for creating the issue, good job!

**Comment by absurd-turtle** [comment]
I'd suggest to add a field called 'common_name_de' as we do not intend to add more languages and this is the easiest solution.
As far as I know the backend does not know about the language settings in the frontend.
We can then simply decide which language to display in the frontend.

**Comment by markus2330** [comment]
Yes, I agree. I added this decision (language concept) also to 403bb67b92f32d09c9651433c6c636f68911e646

**Comment by markus2330** [comment]
> Should English common names be used as a fallback in the case that the language is set to German but no German common names are available for a plant?

Thx for telling! The fallback is fine for now.

But there are other problems which are now unveiled (probably existed before but we didn't see them as there were no German plant names): #1249
